lib/models/student.js

```diff
@@ -42,6 +42,9 @@ module.exports = class Student extends Model {
             percentageIn12th: Joi.string().allow(null),
             mathMarksIn12th: Joi.string().allow(null),
 
+            // Record students type of test
+            typeOfTest: Joi.string().allow(null),
```

Comment:
Don't Allow the typeOfTest to be null it should be either offline or online

Reply:
You can create an enum in constant.js

Reply:
it is done.